src/main/java/org/folio/rest/repository/FeeFineActionRepository.java

```diff
@@ -180,6 +182,16 @@ private GroupedCriterias getTypeCriteria(Action action) {
       .collect(Collectors.toList());
   }
 
+  private List<Criteria> getAccountIdCriterias(Set<String> accountIds) {
+    return accountIds.stream()
+      .map(accountId -> new Criteria()
+        .addField("'accountId'")
+        .setOperation("=")
+        .setVal(accountId)
+        .setJSONB(true))
+      .collect(Collectors.toList());
+  }
+
   private GroupedCriterias groupCriterias(List<Criteria> criterias, String op) {
```

Comment:
Can this method be refactored into a common utility class of some sort so that it can be reused by AccountRepository.java, which recreates this method for its use?

Reply:
I have already done some refactoring in the code and it was removed